torchvision/transforms/functional.py

```diff
@@ -713,18 +720,36 @@ def rotate(img, angle, resample=False, expand=False, center=None, fill=0):
         center (2-tuple, optional): Optional center of rotation.
             Origin is the upper left corner.
             Default is the center of the image.
-        fill (3-tuple or int): RGB pixel fill value for area outside the rotated image.
-            If int, it is used for all channels respectively.
+        fill (n-tuple or int or float): Pixel fill value for area outside the rotated
+            image. If int or float, the value is used for all bands respectively.
+            Defaults to 0 for all bands. This option is only available for ``pillow>=5.2.0``.
 
     .. _filters: https://pillow.readthedocs.io/en/latest/handbook/concepts.html#filters
 
     """
+    def verify_fill(fill, num_bands):
+        if PILLOW_VERSION < "5.2.0":
```

Comment:
While the optional `fillcolor` became available for `Image.transform` in `pillow==5.0.0`, it was only added to `Image.rotate` in [`pillow==5.2.0`](https://pillow.readthedocs.io/en/latest/releasenotes/5.2.0.html#api-additions).